packages/mdc-theme/README.md

```diff
@@ -85,8 +89,12 @@ CSS Class | Description
 `mdc-theme--primary` | Sets the text color to the theme primary color
 `mdc-theme--secondary` | Sets the text color to the theme secondary color
 `mdc-theme--background` | Sets the background color to the theme background color
+`mdc-theme--error` | Sets the error color to the theme error color
```

Comment:
You have surface before error above; want to do the same here and in instances below?